main.c

```diff
@@ -33,18 +36,30 @@ int Init() {
 
     printf("Welcome to Tic Tac Toe! Copyright (C) 2021\n");
 
-    // On success this returns 0
+    // On success these return 0
     if(SDL_Init(SDL_INIT_VIDEO | SDL_INIT_GAMECONTROLLER) != 0) {
-        printf("Couldn't initialize SDL! Reason: %s", SDL_GetError());
+        printf("Couldn't initialize SDL! Reason: %s\n", SDL_GetError());
     }
 
+    if(TTF_Init() != 0) {
```

Comment:
Maybe this can be added to `screenInit()` (and maybe renaming it to `graphicInit()` or similar?)

Reply:
One could also think about moving the SDL_Init as well

Reply:
I believe it's better to keep it in Init() after the copyright message and all.